app/services/media-gallery/stock-library.ts

```diff
@@ -0,0 +1,321 @@
+    fileName: 'FNQ5CFj.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/fc3l87Y.gif',
+    fileName: 'fc3l87Y.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/N3qEsr2.gif',
+    fileName: 'N3qEsr2.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/IprghJf.gif',
+    fileName: 'IprghJf.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/DAFaUJS.gif',
+    fileName: 'DAFaUJS.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/TPDLKu1.gif',
+    fileName: 'TPDLKu1.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/xbzwN0P.gif',
+    fileName: 'xbzwN0P.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/OFEoWbv.gif',
+    fileName: 'OFEoWbv.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/jx9DtIK.gif',
+    fileName: 'jx9DtIK.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/rS0A2Zx.gif',
+    fileName: 'rS0A2Zx.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/nYO9x0u.gif',
+    fileName: 'nYO9x0u.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/hMLMJYs.gif',
+    fileName: 'hMLMJYs.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/VaS6lli.gif',
+    fileName: 'VaS6lli.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/eA9vnyJ.gif',
+    fileName: 'eA9vnyJ.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/5bWLiWD.gif',
+    fileName: '5bWLiWD.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/5cY9mqW.gif',
+    fileName: '5cY9mqW.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/bZIQIad.gif',
+    fileName: 'bZIQIad.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/hjajnLq.gif',
+    fileName: 'hjajnLq.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/jLbvtgH.gif',
+    fileName: 'jLbvtgH.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/D1Lz6qV.gif',
+    fileName: 'D1Lz6qV.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/1n9bK4w.gif',
+    fileName: '1n9bK4w.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/9NDQVfz.gif',
+    fileName: '9NDQVfz.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/Mn4Shfh.gif',
+    fileName: 'Mn4Shfh.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/SkCa8fk.gif',
+    fileName: 'SkCa8fk.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/xGTaKmK.gif',
+    fileName: 'xGTaKmK.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/dBXACr6.gif',
+    fileName: 'dBXACr6.gif'
+  },
+  {
+    href: 'http://uploads.twitchalerts.com/image-defaults/sJJBQOT.gif',
+    fileName: 'sJJBQOT.gif'
+  }
+];
```

Comment:
I removed the `map` loops from this file. The potential issue here - the code in such files will be executed on application startup even if you are not going to open the gallery. This is not a big problem for one file but, it can affect the application starting time if we will have many files like this. I moved `map` loops into the service here. Our services have lazy initialization, and services initialize the data only when is your first time of using it.

Reply:
that's totally fine by me, that was just copypasted from the website